src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -644,45 +607,25 @@ const promise = makeAPICall();
 API.cancel(promise, "my error message");
 ```
 
-### Recap
-You have implemented a few mutations to change your data. 
-
-[comment]: # (ADD TO RECAP a tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
-[comment]: # (RENE, more detail needed here? We're not really addressing the tie-in or real-world use case or the benefit of what we've done in this module.)
+**Recap:** You have implemented a few mutations to change your data. 
```

Comment:
More detail needed here? We're not really addressing the tie-in or real-world use case or the benefit of what we've done in this module.

Reply:
You are now able to create, update, and delete your app data via the GraphQL API.